### ISSUE TITLE
GlobalAvgPool2d causes the inconsistency of output between frameworks

### ISSUE THREAD

**Issue by Crystal-yyy** [pytorch/pytorch#76956]
### 🐛 Describe the bug

Operators with GlobalAvgPool2d will cause inconsistency between frameworks. And for this inconsistency, only changing the input may cause its nature to change.
（Inconsistency: When testing the three frameworks, the inconsistency here refers to comparing the output of the GlobalAvgPool2d layer with the other two frameworks and computing their absolute values.）

**structure：**
`[{'name': 'MaxPool2d', 'params': {'pool_size': [3, 2], 'stride': [4, 3], 'padding': 'same'}}, {'name': 'GlobalMaxPool2d', 'params': {}}, {'name': 'Dense', 'params': {'in_features': 1, 'out_features': 10}}, {'name': 'Softmax'}]`

### Versions

Ubuntu 20.04.2 LTS (GNU/Linux 5.4.0-74-generic x86_64)
Pytorch1.8
gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0
NVIDIA-SMI 460.80 Driver Version: 460.80 CUDA Version: 11.2
